### DISCUSSION TITLE
自由是有限度的，所以权力不是更应该关在笼子里吗？

### DISCUSSION THREAD

**Discussion by yalo-ddaa** [WeNeedHome/SummaryOfLoanSuspension#479] [Q&A]
无论在哪里，自由永远都是有限度的

经常有人说自由是有限度的，这句话本身没有问题。
但为什么没有人说权力的自由也是有限度的？
凭什么权力机关想不让你取钱就可以不让你取钱，想赋红码就可以赋红码？

**普通人的自由有限，那官老爷们的自由是无限的吗？**

**Comment by zxcs001** [comment]
对的，就是无限的。

因为国人们长久的纵容，求锤得锤。什么样得人配得上什么样得政府。

你去看看疫情初往国内得捐款，再看看去年河南大水、今年广东大水海外捐款就知道。

现在很多海外华人看到国内韭菜被锤，都是看笑话，幸灾乐祸得角度。看到某些国内底层失业，生活困苦得视频，下面基本上都是调侃得，什么开香槟的，同情和关切的评论从20年初占大多数，变成现在的大多数人都在嘲讽，少数人阴阳怪气。我是一点一点的看过来的。

**Reply by yalo-ddaa** [reply]
舆论还是有引导的

**Reply by zxcs001** [reply]
> 舆论还是有引导的

对的，国内疫情初期来个千里投毒。把所有海外华人推向对立面这种做法，让留学生和海外华人背锅的做法，真是让人佩服，还有比这帮内宣更猪头的人么？还有那个坚子，更是牛逼，外交部不是交朋友的，是把自己敌人搞多一些。

这舆论只用来洗脑韭菜们，真是有用啊。

这几个月连韭菜都快忽悠不住了，抖音，西瓜视频，头条这种韭菜集散地，各种评论翻车。舆论吹的再牛逼，到被裁员，失业时候也不管用了。

**Reply by Crazy-Creeper** [reply]
所以说目前我国主要矛盾是人民日益增长的智商和政府落后的欺诈技巧

**Comment by xiaer93** [comment]
这也是我所关注的，我同意自由是有限的，但是 言论自由的边界在哪里？似乎 **生活中处处是红线**。

1. 短视频中各种拼音缩写，评论留言敏感词发送失败。
2. 微博降热度、文章删帖、404。
3. 7.10警民打在了一起，没有一家媒体敢报道。
4. 在我的日常生活中，没有人讨论国内时政，大家都知道“查水表”。

**Reply by yalo-ddaa** [reply]
确实。

**Comment by liuwenwu520** [comment]
你们说的问题确实，没有带头的人能够反抗当前这种情况，国内现在的媒体都只报喜不报忧，就这个停贷的很多都被删除了，不允许上传等等

**Comment by ghost** [comment]
完整的社会是一个平衡的艺术，单纯把政治的权力关在笼子里，难道要让资本来管理国家吗？
没有完整可靠的理论和实践验证就来指点江山，谁给你的自信？？？

**Reply by yalo-ddaa** [reply]
然而在我国，权力和资本基本是一体的。我国不存在真正的资本家。

**Comment by yjeff112** [comment]
原来是你小子想断老爷们财路？

**Comment by TALK1949** [comment]
通常这么说的人只是想让你闭嘴，他们自己都不承认有绝对的自由，却总是喜欢把别人阐述的自由“绝对化”偷换概念在反驳对方，逻辑都没法自洽。
